shacl/register.shacl.ttl

```diff
@@ -0,0 +1,198 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix schema: <http://schema.org/> .
+@prefix dcat: <http://www.w3.org/ns/dcat#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix dc: <http://purl.org/dc/terms/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix reg: <http://terms.netwerkdigitaalerfgoed.nl/ns/register#> .
+
+##
+## Schema.org
+##
+
+#
+# Class shapes
+#
+
+reg:DatacatalogShape
+	a sh:NodeShape ;
```

Comment:
Lots of tabs while the whole repository is space-based. Please use spaces for now, I’ll push an editorconfig to the main branch later to fix any remaining formatting issues.